pkg/helpers/newapp/scheme.go

```diff
@@ -0,0 +1,30 @@
+package newapp
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+
+	"github.com/openshift/api"
+	"github.com/openshift/api/route"
+
+	schemehelper "github.com/openshift/oc/pkg/helpers/scheme"
+)
+
+var NewAppScheme = runtime.NewScheme()
+
+var NewAppCodecs = serializer.NewCodecFactory(NewAppScheme)
+
+func InstallSchemes() {
```

Comment:
instead of having an init function here?

the advantages for having it here:
- contains the logic in a single place
- simplify the dependency graph between the packages

Reply:
You are right, I updated.